news/_posts/core-weekly/2022-07-25-coreweekly-29-2022.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 29 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-07-25
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 18th to Sunday 24th of July 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+[write a nice message here, or remove the "General messages" section]
```

Comment:
Maybe comment on today's release? Status update for v8?

Reply:
Yes I waited purposefully on 26/07/2022 to be able to remind informations about security issue